src/renderer/coremods/contextMenu/index.tsx

```diff
@@ -78,7 +85,7 @@ export function removeContextMenuItem(navId: ContextMenuTypes, getItem: GetConte
 }
 
 type ContextMenuData = ContextMenuProps["ContextMenu"] & {
-  children: React.ReactElement | React.ReactElement[];
+  children: React.ReactElement[];
```

Comment:
Why?

Reply:
because I make it an array if it's just a single element anyway.

Reply:
But then users can't input a single element

Reply:
- they can? wdym, this is type of what we get from api not anything users input